source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -0,0 +1,65 @@
+title: Check the location of your ``.so`` files
+ref: check-location
+level: 4
+content: |
+  The |odbc-driver-name| files are named ``libmdbodbca.so`` and
+  ``libmdbodbcw.so``. After downloading the |odbc-driver-name|,
+  move them to your ``/usr/local/lib`` directory.
+---
+title: Create an ``odbc.ini`` file
+ref: create-odbc-ini
+level: 4
+content: |
+  The file ``/etc/odbc.ini`` defines your Data Source Names. Below is
+  an example ``odbc.ini`` file which is configured to use the
+  |odbc-driver-name| with a DSN called ``MongoDBODBC``. Replace all
+  necessary placeholder values with values for your system.
+
+  The file named ``libmdbodbca.so`` is the ANSI driver, and the
+  file named ``libmdbodbcw.so`` is the Unicode driver. Specify
+  the desired driver in your ``odbc.ini`` file.
```

Comment:
Consider changing this to: "Specify the desired driver by setting the ``Driver`` setting in your ``odbc.ini`` file." or something like that

Reply:
Done.